paper_io_sim.tex

```diff
@@ -748,6 +760,9 @@
     \section{Results}
     \label{results}
 
+
+    \todo[inline]{HC: Not sure what "Absolute relative simulation errors" means in the caption of Figure~\ref{fig:single_error}. I assume it's a typo and you just mean "Relative"?}
+
```

Comment:
What is shown is the absolute value of the relative error. Shall I rename to "Relative simulation errors (absolute values)"?

Reply:
ok great. Last suggestion on this figure. Maybe we could cap the y axis at 300%  for both graphs? So that the "WRENCH"bar is reaches all the way to the top, when it's over 300%, but we get to see the lower error bars a bit better? Or perhaps visually you like it better that way (so that the errors "look" smaller relative to the purple WRENCH bars).  more importantly, we need a legend that shows that light blue is Prototype, Purple is WRENCH, and dark blue is WRENCH-cache. It's weird because in the result/single/plot_error.py script I see the legend code, but it seemed it doesn't get produced in the figure result/single/figures/single_errors_v2.pdf .  Perhaps I am doing something wrong?

Reply:
Point was to have the same y-scale on both graphs, as they are side by side. It makes it easier to compare the errors between the two file sizes. I'll fix the legend, not sure why it disappeared :sweat:

Reply:
but then for 20GB WRENCH (purple bars) would be truncated, do we want that?

Reply:
that was my idea. "above 300% error" just means "super inaccurate", so who cares what the data is. And in the text we could just say (it goes up to about 600%).  Again, I don't feel super strongly about this. Or perhaps cap at 400% so that no purple bar gets truncated for the figure on the right. The other option is a log scale of course. It's just that as a reader, what I care about is the height of the blue bars, and at that scale I can't read the values at all besides "under 100%".

Reply:
I agree that we can't see anything about the blue bars. I'm still not super comfortable truncating the purple bars thoough but I like the logscale, what do you think?

![single_errors](https://user-images.githubusercontent.com/5174953/119059818-0f25b480-b99f-11eb-8938-b627df5a894b.png)

Reply:
I like it!

Reply:
done deal then!